.eslintrc.js

```diff
@@ -10,5 +10,9 @@ module.exports = {
 			'estree'
 		],
 		'svelte3/compiler': require('./compiler')
+	},
+	// workaround for https://github.com/typescript-eslint/typescript-eslint/issues/1824 and rely solely on 'indent' rule
+	rules: {
+		"@typescript-eslint/indent": "off"
 	}
```

Comment:
prior to making this change, it was not possible to use `--fix` to resolve the lint warnings reported by 'indent' itself.

this leads me to believe that 'indent' and '@typescript-eslint/indent" are at an impasse as to what the appropriate indentation should be. given the language in https://github.com/typescript-eslint/typescript-eslint/issues/1824, it seemed safest to side with 'indent'.